### DISCUSSION TITLE
Allow only x # of CODEOWNERS to approve a PR

### DISCUSSION THREAD

**Discussion by ts46236** [community/community#12844] [Pull Requests]
We want to require at least 1 code owner but the way it works, all code owners must review which is not very workable

**Comment by turadg** [comment]
Does assigning a team as the owner solve this for you?

**Comment by LosAhuertos** [comment]
вс, 1 мая 2022 г., 16:36 Turadg Aleahmad ***@***.***>:

**Comment by ts46236** [comment]
When you say team, are you referring to a team distribution list (email group) or a system group (in my case that would be Active Directory)?

**Reply by turadg** [reply]
[Github team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams)
